src/backend/empty/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "empty"
+version = "0.1.0"
+description = "Empty backend for gfx-rs"
+publish = false
+workspace = "../../.."
+
+[lib]
+name = "empty"
+
+[dependencies]
+gfx_core = { path = "../../core", version = "0.7" }
```

Comment:
I'm not totally sure if moving dummy out works as it's required for the docs of `core`.
Have you tested it?

Reply:
Would be nice to keep it the current way to keep `core` small and be a bit more consitent.
I'm currently trying out if this is possible.